site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -2,4 +2,82 @@
 api: runtime
 ---
 
-<!-- Intentionally blank -->
+## Examples
+
+### Use [`getURL`][get-url] to add an extension image to a page {: #example-get-url }
+
+In order for a web page to access an asset hosted on another domain, it must specify the resource's
+full URL (e.g. `<img src-"https://example.com/logo.png">`). The same is true for when a web page
+wants to include assets included in an extension. The two main differences here are that the
+extension's assets must be exposed as [web_accessible_resources][war] and that typically content
+scripts are responsible for injecting extension assets.
+
+This example shows how a [content script][content] can add an image in the extension's package to
+the page that the content script has been [injected][content-inject] into.
+
+```js
+//// content.js
+
+{ // Block used to avoid setting global variables
+  let img = document.createElement('img');
+  img.src = chrome.runtime.getURL('logo.png');
+  document.body.append(img);
+}
+```
+
+### Getting background data into a content script {: #example-content-msg }
+
+It's common for an extension's content scripts to need data managed by another part of the
+extension, like the extension's background script. Much like two browser windows opened to the same
+web page, these two contexts cannot directly access each other's values. Instead, the extension can
+use [message passing][message-passing] to coordinate across these different contexts.
+
+In this example, the content script needs some data from the extension's background script in order
+to initialize it's UI. To get this data, it passes a `get-user-data` message to the background, and
+the background responds with a copy of the user's information.
```

Comment:
@awfuchs, I rewrote the intro to this section. Let me know what you think.

Reply:
@awfuchs indicated offline that he has some nits, but will address them in another PR he's planning to work on for this page.